tpl/template_embedded.go

```diff
@@ -233,7 +233,7 @@ func (t *GoHTMLTemplate) EmbedTemplates() {
 {{ end }}{{ end }}
 
 <!-- Output all taxonomies as schema.org keywords -->
-<meta itemprop="keywords" content="{{ range $plural, $terms := .Site.Taxonomies }}{{ range $term, $val := $terms }}{{ printf "%s," $term }}{{ end }}{{ end }}" />
+<meta itemprop="keywords" content="{{ if .IsPage}}{{ range $index, $tag := .Params.tags }}{{ $tag }},{{ end }}{{ else }}{{ range $plural, $terms := .Site.Taxonomies }}{{ range $term, $val := $terms }}{{ printf "%s," $term }}{{ end }}{{ end }}{{ end }}" />
```

Comment:
@brunoamaral here's what I got so far regarding a page-level meta, in case it helps some WRT slicing and dicing for Schema: https://github.com/comfusion/after-dark/commit/4b1f0f1ae2785fc636d84380a8f803d0e689f5e7